README.md

```diff
@@ -121,6 +123,55 @@ In addition, the following ports are provided in this repository:
  - [webassembly](ports/webassembly) -- Emscripten port targeting browsers and NodeJS.
  - [zephyr](ports/zephyr) -- Zephyr RTOS.
 
+Port Tier levels
+----------------
+
+MicroPython operates with a set of Tier levels for the ports. Tiers 1 through 3
+are for ports under active development.  Tier M is for older ports now in
+maintenance mode.  Tier T is for ports used primarily for testing. These Tier
+levels are defined in the table below.
+
+|                                     | Tier 1 | Tier 2 | Tier 3 | Tier M | Tier T |
+|-------------------------------------|--------|--------|--------|--------|--------|
+| actively maintained                 | x      | x      |        | x      | x      |
+| stable Python API                   | x      | x      |        | x      |        |
+| new features actively developed     | x      | x      | x      |        |        |
+| prioritized bug reports             | x      |        |        |        | x      |
+| regressions warrant a patch release | x      |        |        |        |        |
+| builds pass under CI                | x      | x      | x      | x      | x      |
+| tested on hardware for releases     | x      | x      |        | x      |        |
+| tested in CI                        |        |        |        |        | x      |
+
+Tier 1 ports:
+- esp32
+- rp2
+- stm32
+- unix
+
+Tier 2 ports:
+- alif
+- mimxrt
+- nrf
+- renesas-ra
+- samd
+- webassembly
+- windows
+- zephyr
```

Comment:
Zephyr with it#s support for various hardware is more a sub-system than a single port. Something between a pure hardware port like RP2 and a software only like unix.

Reply:
Yes, zephyr is still a port (from MicroPython's point of view).